benchmarks/agieval/agieval_en.py

```diff
@@ -0,0 +1,63 @@
+from agieval import task_template
+
+from inspect_ai import task
+
+
+## English Tasks
```

Comment:
Would it make sense to combine all these tasks into one that takes `dataset_name` as an input argument?